packages/thumbprint-react/components/Image/get-scroll-parent.test.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import getScrollParent from './get-scroll-parent';
+
+test('defaults to `document.body`', () => {
```

Comment:
Maybe add one more test to ensure it works when `document.body` isn't the immediate parent or switch this test to go up more than one level?